TestgenBookshopAPI/service/scripts/build.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# build.sh [repository]
+# Build the Bookshop images and push them to the specified repository.
+# You must already be logged in to the repository.
+# The default repository is the IBM Public Repository.
+
+cd $(dirname $0)
+script="$(basename $0)"
+repository=""
+
+USAGE="Usage:
+  ${script} --repository <repository> --api-key <api-key> [options]
+
+Common options:
+  -h, --help
+"
+
+function usage {
+  local msg="$1"
+  echo && echo "${msg}" >&2
+  echo && echo "${USAGE}" >&2
+  exit 2
+}
+
+while [[ $# > 0 ]]; do
+  case "$1" in
+  -r | --repository)
+    repository=$2
+    shift 2 || usage "Missing repository"
+    ;;
+  --api-key)
+    api_key=$2
+    shift 2 || usage "Missing API key"
+    ;;
```

Comment:
I think we can get rid of these options and make repository a required argument.

Reply:
Done